src/components/shared/DetailsWidget/detailsWidgetConfig.tsx

```diff
@@ -115,7 +115,10 @@ const getDetailItemsMap = (
 ): { [key in Exclude<ActionPageDetails, 'Permissions'>]: DetailItemConfig } => {
   const shortenedHash = getShortenedHash(transactionHash || '');
   const recipientWalletAddress = recipient?.walletAddress;
-  const isSmiteAction = type === ColonyActionType.EmitDomainReputationPenalty;
+  const isSmiteAction = type.includes(
```

Comment:
Why this change? Just curious

Reply:
@ArmandoGraterol To capture motions also